backend/datahubhel/settings.py

```diff
@@ -20,7 +20,7 @@
     VAR_ROOT=(str, default_var_root),
     LOG_ROOT=(str, default_log_root),
     ALLOWED_HOSTS=(list, []),
-    DATABASE_URL=(str, 'postgis:///datahubhel'),
+    DATABASE_URL=(str, os.environ.get('DATABASE_URL') or 'postgis:///datahubhel'),
```

Comment:
The value will be overridden by the env variable anyway. That's the whole point of the `environ` package. This is just a default value, and no need to manually read anything with `os.environ.get`